gossip/simulation/network.go

```diff
@@ -53,53 +57,66 @@ func NewNetwork(nodeCount int) *Network {
 
 	log.Infof("simulating gossip network with %d nodes", nodeCount)
 
-	stopper := stop.NewStopper()
-
-	rpcContext := rpc.NewContext(&base.Context{Insecure: true}, clock, stopper)
-	tlsConfig, err := rpcContext.GetServerTLSConfig()
+	n := &Network{
```

Comment:
All of these changes in gossip/simulation/network.go are just reorg so that I can call `CreateNode` and `StartNode` externally for the gossip bootstrap info persistence test.